docs/build.sbt

```diff
@@ -0,0 +1,10 @@
+// Uses the out of the box generic theme.
+paradoxTheme := Some(builtinParadoxTheme("generic"))
+
+scalaVersion := "2.12.2"
+
+paradoxProperties in Compile ++= Map(
+  "snip.g8root.base_dir" -> "../../../../src/main/g8",
+  "snip.g8src.base_dir" -> "../../../../src/main/g8/src/main/",
+  "snip.g8srctest.base_dir" -> "../../../../src/main/g8/src/test/"
```

Comment:
In akka build we used a bit of relativism still: https://github.com/akka/akka/blob/master/akka-docs/build.sbt#L35

I am not sure about g8 settings. Wouldn't those be set to the absolute value?